packages/console/src/pages/AuditLogDetails/index.tsx

```diff
@@ -22,19 +23,24 @@ import EventIcon from './components/EventIcon';
 import * as styles from './index.module.scss';
 
 const AuditLogDetails = () => {
-  const { logId } = useParams();
+  const { userId, logId } = useParams();
+  const { pathname } = useLocation();
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { data, error } = useSWR<LogDTO, RequestError>(logId && `/api/logs/${logId}`);
+  const { data: userData } = useSWR<User, RequestError>(userId && `/api/users/${userId}`);
+
   const isLoading = !data && !error;
 
+  const backLink = `/${pathname.slice(0, pathname.lastIndexOf('/'))}`;
+  const backLinkTitle = userId ? (
+    <DangerousRaw>{t('log_details.back_to_user', { name: userData?.name ?? '-' })}</DangerousRaw>
```

Comment:
nit: maybe use `user details` or something similar instead of `-` to make it more clear?

Reply:
updated.